model_analyzer/result/result_manager.py

```diff
@@ -320,11 +320,15 @@ def _setup_for_concurrent_analysis(self):
         model_constraints_list = [
             model.constraints() for model in self._config.analysis_models
         ]
+        model_weighting_list = [
+            model.weighting() for model in self._config.analysis_models
+        ]
 
         self._run_comparators = {
             self._concurrent_analysis_model_name:
                 RunConfigResultComparator(
-                    metric_objectives_list=model_objectives_list)
+                    metric_objectives_list=model_objectives_list,
+                    model_weights=model_weighting_list)
```

Comment:
remind me why both the measurement (the thing being compared) and the comparator (the thing doing the comparison) both need the objectives and model weights?  Something about comparing between models in the same measurement vs comparing between measurements?

Reply:
RunConfigResultComarator compares groups of RunConfigMeasurements (which are called RunConfigResults)